plugins/woocommerce/src/Internal/Admin/ReviewsListTable.php

```diff
@@ -217,6 +217,205 @@ public function single_row( $item ) {
 		<?php
 	}
 
+	/**
+	 * Generate and display row actions links.
+	 *
+	 * @see WP_Comments_List_Table::handle_row_actions() for consistency.
+	 *
+	 * @global string $comment_status Status for the current listed comments.
+	 *
+	 * @param WP_Comment $review         The product review or reply in context.
+	 * @param string     $column_name    Current column name.
+	 * @param string     $primary_column Primary column name.
+	 * @return string
+	 */
+	protected function handle_row_actions( $review, $column_name, $primary_column ) {
+		global $comment_status;
+
+		if ( $primary_column !== $column_name || ! $this->current_user_can_edit_review ) {
+			return '';
+		}
+
+		$review_status = wp_get_comment_status( $review );
+
+		$del_nonce     = esc_html( '_wpnonce=' . wp_create_nonce( "delete-comment_$review->comment_ID" ) );
+		$approve_nonce = esc_html( '_wpnonce=' . wp_create_nonce( "approve-comment_$review->comment_ID" ) );
+
+		$url = "comment.php?c=$review->comment_ID";
```

Comment:
In general I'd be in favour of replacing this with `add_query_arg()`, but at minimum I'd `urlencode()` that review ID.